Readability.js

```diff
@@ -732,7 +732,12 @@ Readability.prototype = {
             candidates.push(ancestor);
           }
 
-          ancestor.readability.contentScore += contentScore / (level === 0 ? 1 : level * 2);
+          // Node score divider:
+          // - parent:             1 (no division)
+          // - grandparent:        2
+          // - great grandparent+: 6
+          var scoreDivider = level === 0 ? 1 : level === 1 ? 2 : 6;
```

Comment:
Actually, hang on... this means the scoring is constant for `level > 1`, right? Can we use `level * 3` instead? Does that break other testcases?